### PR TITLE
 fix: set url to null instead of https when it is null

### LINKED ISSUE #5779: Wizard Step 1: Deleting Webinar URL results in error "not valid URL"
It seems it is not possible to delete a webinar or live stream URL. When an organizer tries to delete it and hits "Save" an error message "Not a valid URL" shows up.

Expected: It should be possible to delete these URLS as the field is not required.

Compare https://eventyay.com/events/fa96ae2c/edit/basic-details

Screenshot from 2020-11-27 00-31-25

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5779

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by maze-runnar** [comment]
@iamareebjamal how to allow null in an URL field on the server side?

**Comment by maze-runnar** [comment]
after rebasing current changes from upstream, on migrating it's giving 
`ERROR [root] Error: Can't locate revision identified by '060a77f9a1ea'` on server?

File: app/components/widgets/forms/link-field.ts [hunk 1]
```diff
@@ -63,7 +63,11 @@ export default class LinkField extends Component<Args> {
   setValue(event: { target: HTMLInputElement }): void {
     const text = event.target.value;
     this.value = this.fixValue(text);
-    this.args.onChange(this.finalValue);
+    if (this.value) {
```
Comment (iamareebjamal) [inline app/components/widgets/forms/link-field.ts:66]: this.value ? this.finalValue : this

Comment (maze-runnar) [reply app/components/widgets/forms/link-field.ts:66]: it is showing ``` Type 'this' is not assignable to type 'string'.
        Type 'LinkField' is not assignable to type 'string'.````

**Pull request merged**